lib/dashing/app.rb

```diff
@@ -70,13 +70,23 @@ def authenticated?(token)
   redirect "/" + dashboard
 end
 
+
 get '/events', provides: 'text/event-stream' do
   protected!
   response.headers['X-Accel-Buffering'] = 'no' # Disable buffering for nginx
-  stream :keep_open do |out|
-    settings.connections << out
+  stream do |out|
     out << latest_events
-    out.callback { settings.connections.delete(out) }
+    settings.connections << connection = {out: out, mutex: Mutex.new, terminated: false}
```

Comment:
Yes, it signifies the `out` resource which we are injecting events into inside `send_event`.